.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectCodeStyleConfiguration">

```

Comment:
Did you mean to commit this?

Reply:
no. tell jetbrains to fuck off for me please